src/pages/home/ReportScreen.tsx

```diff
@@ -352,6 +352,7 @@ function ReportScreen({route, navigation}: ReportScreenProps) {
                 policy={policy}
                 parentReportAction={parentReportAction}
                 onBackButtonPress={onBackButtonPress}
+                openParentReportInCurrentTab
```

Comment:
I'm kinda confused. 

`<MoneyRequestHeader>` is used only 2 times in the app: 
 - in `ReportScreen`
 - in `MoneyRequestReportView` - which is the child of `SearchMoneyRequestReportPage`

and this component is basically only ever used for the report that is actually the transaction thread view. So don't we always want to have this flag set to true?